src/test/java/org/eclipse/basyx/testsuite/regression/submodel/metamodel/map/TestSubmodel.java

```diff
@@ -85,4 +87,27 @@ public void testCreateAsFacadePropertyList() {
 	protected ISubmodel getSubmodel() {
 		return submodel;
 	}
+	
+	@Test(expected = IdShortDuplicationException.class)
+	public void checkForExceptionWithDuplicateIdShortInSubmodel() {
+		Map<String, Object> faultySubmodel = createSubmodelWithDuplicateIdShortProperties();
+
+		Submodel.createAsFacade(faultySubmodel);
+	}
+
+	private Map<String, Object> createSubmodelWithDuplicateIdShortProperties() {
+		Property property1 = new Property("testProp", 5);
+		Property property2 = new Property("testProp", 7);
+		
+		Collection<Map<String, Object>> collection = new ArrayList<>();
```

Comment:
You can use Arrays.asList(property1, property2) here

Reply:
Updated the code.